test/connectivity/modules/NetworkController/LinuxTrafficControlController.cs

```diff
@@ -9,16 +9,18 @@ namespace NetworkController
     using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Logging;
 
-    class LinuxFirewallOfflineController : INetworkController
+    class LinuxTrafficControlController : INetworkController
```

Comment:
```suggestion
    class LinuxNetworkController : INetworkController
```

Reply:
It has this name to describe is using Traffic Control for linux

Reply:
ControlController looks odd to you; how about "LinuxTrafficController" or "LinuxNetworkTrafficController"?